src/types/index.ts

```diff
@@ -24,6 +24,10 @@ export interface Networks {
 
 export interface Network {
   name: string;
+  endpoints: {
+    rpc: string;
+    lightClient: WellKnownChain;
```

Comment:
You should now always assume that a light client will exist. 
This should be:
```
    lightClient?: WellKnownChain;
```

Reply:
Nope, we are assuming they exist as all supported chains have a light client

Reply:
Is there a case where they would not exist?

Reply:
> Is there a case where they would not exist?

The only case is either a custom chain that supports staking or in case we do not want to include light client for a known chain (e.g. polkadot) inside the dashboard. This is of course something that can be altered on a later PR (current PR  implementation supports the network UI when light client does not exist [e.g. shows full width button instead of split to 50/50 of RPC/Light client)